services/workflows-service/src/end-user/end-user.schema.ts

```diff
@@ -2,7 +2,7 @@ import { z } from 'zod';
 
 const ActiveMonitoringSchema = z.object({
   type: z.literal('aml'),
-  vendor: z.literal('veriff'),
+  vendor: z.enum(['veriff', 'test']),
```

Comment:
Why are we showing the 3rd party vendor? Is it necessary?

Reply:
It's good to know which vendor provided the AML data. Will help us in the future.